backend/src/test/java/com/library/backend/controller/MealControllerTest.java

```diff
@@ -0,0 +1,175 @@
+package com.library.backend.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.library.backend.model.Meal;
+import com.library.backend.service.MealService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@WebMvcTest(MealController.class)
+class MealControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private MealService mealService;
```

Comment:
I think this is the wrong place to mock something.
In our Controller Tests we want to test the whole Application.
We dont use a mock here, except it is absolutely neccessary. 

Can you update this Test Class please?